backend/functions/src/matching/functions.ts

```diff
@@ -118,9 +115,6 @@ async function makeMatches(courseId: string) {
       memberData: newGroup,
       createTime: new Date(),
       updateTime: new Date(),
-      shareMatchEmailTimestamp: null,
-      checkInEmailTimestamp: null,
-      addStudentEmailTimestamp: null,
```

Comment:
I think you are missing `templateTimestamps` because no empty timestamps object for groups is returned in the response from `/make`

Actually, I see that you just handle the empty timestamps object on the frontend. That works too